extensions/file/src/main/java/org/mule/extension/file/internal/command/AbstractLocalCopyCommand.java

```diff
@@ -47,26 +47,29 @@
    * @param target the path to the target destination
    * @param overwrite whether to overwrite existing target paths
    * @param createParentDirectory whether to create the target's parent directory if it doesn't exists
+   * @param renameTo the new file name, {@code null} if the file doesn't need to be renamed
```

Comment:
just in case, this jdoc is NOT reached by the documentation generator so it's ok to stay technical here